pkg/osquery/tables/xrdb/testdata/example_query.sql

```diff
@@ -0,0 +1,8 @@
+WITH human_accounts AS (
```

Comment:
Do we need this file?

Reply:
(I'm happy to keep it as an example)

Reply:
Don't need it, I just had it saved to make it easier to run a semi-automatic test during dev. I figured it'd be nice for an example.

Reply:
WFM